src/catalog/CatalogPage.tsx

```diff
@@ -44,40 +47,67 @@ const CatalogPage = () => {
   const totalCourses = courseData?.results?.length ?? 0;
 
   return (
-    <Container className="container-xl pt-5.5">
-      <SubHeader title={intl.formatMessage(messages.totalCoursesHeading, {
-        totalCourses,
-      })}
+    <Container className="container-xl pt-5.5 mb-6">
+      <SubHeader
+        title={intl.formatMessage(messages.exploreCourses)}
```

Comment:
[inform]: Changing the title according to search parameters (legacy functionality) will be added in future releases.